src/Node/FS.purs

```diff
@@ -7,6 +17,32 @@ import Prelude
 --
 foreign import data FS :: !
 
+foreign import data FileDescriptor :: *
+
+data FileFlags = R | R_PLUS | RS | RS_PLUS
+               | W | WX | W_PLUS | WX_PLUS
+               | A | AX | A_PLUS | AX_PLUS
+
+instance showFileFlags :: Show FileFlags where
```

Comment:
>  Show instance produces code that can be evaluated to give back the original value.

The existing purescript-node-fs codebase doesn't do this - consider `showSymLinkType`. 

I don't see much value in requiring shows output to be "evaluated". As long as read and show are symmetric I'm happy.